packages/server/src/notification/desktop-notif-subscriber.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  EventSubscriber,
+  EntitySubscriberInterface,
+  Connection,
+  InsertEvent,
+} from 'typeorm';
+import { DesktopNotifModel } from './desktop-notif.entity';
+import { NotificationService } from './notification.service';
+
+@EventSubscriber()
+export class DesktopNotifSubscriber
+  implements EntitySubscriberInterface<DesktopNotifModel> {
+  notifService: NotificationService;
+  constructor(connection: Connection, notifService: NotificationService) {
+    this.notifService = notifService;
+    connection.subscribers.push(this);
+  }
+
+  listenTo() {
+    return DesktopNotifModel;
+  }
+
+  async afterInsert(event: InsertEvent<DesktopNotifModel>) {
+    console.log(`BEFORE DESKTOPNOTIF INSERTED: `, event.entity);
```

Comment:
Not sure if this console.log is meant for debugging or logging, if it's not logging let's remove it